frontend/packages/console-shared/src/components/formik-fields/InputField.tsx

```diff
@@ -11,12 +17,29 @@ const InputField: React.FC<InputFieldProps> = ({
   onChange,
   validated,
   helpTextInvalid,
+  inputGroupText,
```

Comment:
Adding just a string prop for input group text doesn't really cover all of the scenarios an `InputGroup` might be used. We have so many patterns that are supported by an `InputGroup` https://www.patternfly.org/v4/documentation/react/components/inputgroup. But all these patterns are not supported by this `FormikField`. 

It would be better if you create another field called `InputGroupField` that handles all these scenarios. Also look at the implementation of `CheckBoxField` and `SwitchField` that use the same logic from `ToggleableFieldBase`. `InputField` and `InputGroupField` can follow similar logic of code sharing.